{"name": "test_sending_email_wrong_recepient", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <test_project.tests.send_email_test.TestSendEmail object at 0x7f8c3a782ba8>\n\n    def test_sending_email_wrong_recepient(self):\n        login = 'pasekpasek61@gmail.com'\n        password = 'Pasek61pasek'\n    \n        recipient = \"lama@\"\n        title = LoremIpsum(driver=self.driver).generate_lorem_ipsum(3)\n        content = \"alin1234\"\n    \n        log_in_page = GmailLogIn(driver=self.driver)\n        wait = log_in_page.wait\n        logger = log_in_page.logger\n    \n        log_in_page.open_main_page()\n        log_in_page.input_login(login)\n        log_in_page.input_password(password)\n        send_mail_page = GmailSendMail(wait, logger)\n        send_mail_page.send_mail(recipient, title, content)\n    \n>       assert send_mail_page.get_error_wrong_recepient()\n\ntest_project/tests/send_email_test.py:69: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntest_project/pages/send_email.py:56: in get_error_wrong_recepient\n    return self.wait.until(EC.visibility_of_element_located((By.XPATH, message_wrong_recepient_xpath)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"43762099-2c2c-434f-82ee-ec35dcc709ec\")>\nmethod = <selenium.webdriver.support.expected_conditions.visibility_of_element_located object at 0x7f8c3a788898>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/usr/lib/python3/dist-packages/selenium/webdriver/support/wait.py:80: TimeoutException"}, "steps": [{"name": "Opening page", "status": "passed", "start": 1567944239167, "stop": 1567944259489}, {"name": "Setting login", "status": "passed", "parameters": [{"name": "login", "value": "'pasekpasek61@gmail.com'"}], "start": 1567944259490, "stop": 1567944261714}, {"name": "Setting password", "status": "passed", "parameters": [{"name": "password", "value": "'Pasek61pasek'"}], "start": 1567944261716, "stop": 1567944322236}, {"name": "Setting recipient of mail: ''lama@'', title: ''Curabitur ut fringilla.'' and content:'alin1234'", "status": "passed", "parameters": [{"name": "recipient", "value": "'lama@'"}, {"name": "title", "value": "'Curabitur ut fringilla.'"}, {"name": "content", "value": "'alin1234'"}], "start": 1567944322239, "stop": 1567944332156}, {"name": "Checking if the message about the invalid recipient is displayed", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"/usr/local/lib/python3.7/dist-packages/allure_commons/_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"/home/alina/PycharmProjects/gmail3/test_project/pages/send_email.py\", line 56, in get_error_wrong_recepient\n    return self.wait.until(EC.visibility_of_element_located((By.XPATH, message_wrong_recepient_xpath)))\n  File \"/usr/lib/python3/dist-packages/selenium/webdriver/support/wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1567944332157, "stop": 1567944437571}], "start": 1567944216558, "stop": 1567944437574, "uuid": "d8cc122a-2021-4645-a3d4-e68693cb52bf", "historyId": "41b6235d9b00847beefc77ff7c5818b2", "fullName": "test_project.tests.send_email_test.TestSendEmail#test_sending_email_wrong_recepient", "labels": [{"name": "parentSuite", "value": "test_project.tests"}, {"name": "suite", "value": "send_email_test"}, {"name": "subSuite", "value": "TestSendEmail"}, {"name": "host", "value": "alina-Inspiron-N5050"}, {"name": "thread", "value": "9198-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_project.tests.send_email_test"}]}